e2etest/rewarding_test.go

```diff
@@ -159,6 +156,15 @@ func TestBlockEpochReward(t *testing.T) {
 
 	// Set mini-cluster configurations
 	rand.Seed(time.Now().UnixNano())
+	var delegates []genesis.Delegate
+	for i := 0; i < numNodes; i++ {
+		delegates = append(delegates, genesis.Delegate{
+			OperatorAddrStr: identityset.Address(i + numNodes).String(),
+			RewardAddrStr:   identityset.Address(i).String(),
```

Comment:
1. rewardAddr and operatorAddr got reversed
2. VotesStr = strconv.Itoa(1000 + rand.Intn(1000))